pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -36,7 +36,7 @@ import (
 	"github.com/lf-edge/eve/pkg/pillar/sema"
 	"github.com/lf-edge/eve/pkg/pillar/types"
 	"github.com/lf-edge/eve/pkg/pillar/wrap"
-	"github.com/satori/go.uuid"
+	uuid "github.com/satori/go.uuid"
```

Comment:
Why change this? Not needed and might make others think we need the same or similar changes in other files.

Reply:
I haven't modified this, got changed automatically when built.
The next line of code on line 40 has similar change
log "github.com/sirupsen/logrus"

I will remove it manually

Reply:
FWIW The line for logrus has always been that way with "log" up front.